beater/bundle/bundle.go

```diff
@@ -42,7 +42,7 @@ var (
             }
         },
         "decision_logs": {
-            "console": true
+            "console": false
```

Comment:
The decision logs are not affected by the logger changes that are part of the PR.
IMO it's OK to disable the decision logs to the console since we already have the decisions printed as a debug log in `events_creator.go`.
WDYT @amitkanfer @eyalkraft ?

Reply:
I agree.